code/modules/customitems/item_defines.dm

```diff
@@ -2038,4 +2038,231 @@ obj/item/clothing/suit/storage/hooded/fluff/make_poncho //Raincoat Poncho - M.A.
+	name = "clan marwani armor"
+	desc = "A large, decorative iron and brass alloyed breastplate signifying the industrial strength of Clan Marwani."
+	icon = 'icons/obj/custom_items/marwani_dress.dmi'
+	icon_state = "mar_emblem"
+	item_state = "mar_emblem"
+	contained_sprite = TRUE
+
+
+/obj/item/weapon/deck/tarot/fluff/klavdiya_cards //Adhomian Divination Cards Deck - Klavdiya Tikhomirov - alberyk
+	name = "adhomian divination cards deck"
+	desc = "An adhomian deck of divination cards, used to read the fortunte or play games."
+	icon_state = "deck_adhomai"
+
+/obj/item/weapon/deck/tarot/fluff/klavdiya_cards/generate_deck()
+	var/datum/playingcard/P
+	for(var/name in list("D'as'ral Massacre","Clairvoyant","Suns' Sister","Queen","King","Father of the Parivara","S'rendal'Matir","Tank","Royal Grenadier","Kraszarrumalkarii","Hand of Fate","Great Revolution","Assassin","Assassination","Dymtris Line",
+	"Rrak'narrr","Steeple","Messa","Raskara","S'rendarr","Kazarrhaldiye","Adhomai"))
+		P = new()
+		P.name = "[name]"
+		P.card_icon = "adhomai_major"
+		P.back_icon = "card_back_adhomai"
+		cards += P
+	for(var/suit in list("wands","pentacles","cups","swords"))
+
+
+		for(var/number in list("ace","two","three","four","five","six","seven","eight","nine","ten","serf","soldier","queen","king"))
+			P = new()
+			P.name = "[number] of [suit]"
+			P.card_icon = "adhomai_[suit]"
+			P.back_icon = "card_back_adhomai"
+			cards += P
+
+/obj/item/clothing/accessory/locket/fluff/klavdiya_amulet //Moon Shaped Amulet - Klavdiya Tikhomirov - alberyk
+	name = "moon shaped amulet"
+	desc = "A metalic necklace that bears a silver moon shapped pendant."
+	icon = 'icons/obj/custom_items/klavdiya_amulet.dmi'
+	icon_state = "klavdiya_amulet"
+
+
+/obj/item/clothing/gloves/fluff/lunea_gloves //Spark Gloves - Lunea Discata - tishinastalker
+	name = "spark gloves"
+	desc = "Custom made flame retardant gloves designed after Cpt. Stallion from hit 2451 anime series Truesteel Arcanist: Fellowship. \
+	A lighter assembly is built along the index finger with a tiny plate of steel, and there is a small flint built into the thumb."
+	icon = 'icons/obj/custom_items/lunea_gloves.dmi'
+	icon_state = "lunea_gloves"
+	item_state = "lunea_gloves"
+	contained_sprite = TRUE
+	var/lit = FALSE
+
+/obj/item/clothing/gloves/fluff/lunea_gloves/verb/toggle()
+	set name = "Toggle Spark Gloves"
+	set category = "Object"
+	set src in usr
+
+	if (use_check(usr)) return
+
+	if(!lit)
+		usr.visible_message("<span class='notice'>With a snap of \the [usr]'s fingers, a small lighter flame sparks from \his index fingers!</span>")
+		lit = TRUE
+		playsound(src.loc, 'sound/items/lighter_on.ogg', 75, 1)
+		update_icon()
+		usr.update_inv_gloves()
+		return
+
+	else
+		usr.visible_message("<span class='notice'>With a flick of \the [usr]'s wrists and a close of \his fingers, the small flames are extinguished.</span>")

```

Comment:
Would better verbage be "With the flick of [user] wrists and the pinch of their fingers, the glove's flames are extinguished."